libnetwork/resolver_unix.go

```diff
@@ -4,102 +4,59 @@
 package libnetwork
 
 import (
-	"fmt"
 	"net"
-	"os"
-	"os/exec"
-	"runtime"
 
 	"github.com/docker/docker/libnetwork/iptables"
-	"github.com/docker/docker/pkg/reexec"
 	"github.com/sirupsen/logrus"
-	"github.com/vishvananda/netns"
 )
 
-func init() {
-	reexec.Register("setup-resolver", reexecSetupResolver)
-}
-
 const (
 	// outputChain used for docker embed dns
 	outputChain = "DOCKER_OUTPUT"
 	//postroutingchain used for docker embed dns
```

Comment:
Not directly related, but we should fix the formatting of this comment (perhaps renamed the const to `postRoutingChain` as well to be more ok(ish) camelCase

```suggestion
	// postroutingchain used for docker embed dns
```

Reply:
Will do in #44664